containers/proxy-helm/templates/ingress.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  creationTimestamp: null
```

Comment:
A nitpick, is this needed?

Reply:
No it's not, Just that `kubectl` commands add it. I'll remove it